mwdb/web/src/components/ShowObject/Views/CommentBox.js

```diff
@@ -12,7 +12,7 @@ import { updateActivePage } from "@mwdb-web/commons/helpers";
 
 function Comment(props) {
     return (
-        <li className="media" style={{ wordBreak: "break-all" }}>
+        <li className="media" style={{ wordWrap: "break-word" }}>
```

Comment:
Hmm, I don't think it's good solution:
<img width="889" alt="image" src="https://user-images.githubusercontent.com/8720367/146934015-0641d093-e886-4617-98bf-14c58d3f8744.png">

1. `word-wrap` is deprecated and now it's called `overflow-wrap` (https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap). As we don't target IE, we should use current property name (https://caniuse.com/wordwrap)
2. As in MDN: In contrast to `word-break`, `overflow-wrap` will only create a break if an entire word cannot be placed on its own line without overflowing. We still want to break too long words in the middle to prevent overflowing. So instead of `word-break: break-all` we should use `overflow-wrap: anywhere`

<img width="362" alt="image" src="https://user-images.githubusercontent.com/8720367/146934610-f5ebe280-9d0e-4de3-97e0-066e9a23ef22.png">

```suggestion
        <li className="media" style={{ overflowWrap: "anywhere" }}>
```

Reply:
You are right. Thanks for correction. Update to PR was sent.